app/components/date_single/edit_component.html.erb

```diff
@@ -0,0 +1,29 @@
+<%= tag.div data: { controller: 'form-date' }, class: reset_class  do %>
```

Comment:
Previously PublicationDate::Edit but now used for other dates.

Reply:
@justinlittman can you explain the "date_single" naming here? Thank you.

Reply:
At one point the name made sense; it no longer does. Renaming.